numalogic/models/vae/variants/conv.py

```diff
@@ -101,7 +99,7 @@ def forward(self, x: Tensor) -> tuple[Tensor, Tensor]:
 
 class Decoder(nn.Module):
     """
-    Decoder (non-probabilistic) module for Convolutional Variational Autoencoder.
+    Decoder module for Convolutional Variational Autoencoder.
```

Comment:
No, it is not equivalent. Padding is mostly done with zeros in Conv layers. Padding is done so that the values at the edge of the sequence have the same weightage as the non-edged data points.